update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -51,7 +51,11 @@ public static void main( String[] args ) {
       authorIdTR = Integer.valueOf(props.getProperty("authorIdTR"));
       authorIdGK = Integer.valueOf(props.getProperty("authorIdGK"));
       int port = Integer.valueOf(props.getProperty("port"));
-      testMode = Boolean.valueOf(props.getProperty("testMode"));
+      if (props.getProperty("testMode") == null) {
+        testMode = true;
+      } else {
+        testMode = Boolean.valueOf(props.getProperty("testMode"));
```

Comment:
Since `testMode` is initialized to true, could this be:
```
if (props.getProperty("testMode") != null) {
    testMode = Boolean.valueOf(props.getProperty("testMode"));
}
```